### PR TITLE
OPENEUROPA-721: Update code-review and grump.yml.dist file.

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "phpunit/phpunit": "^5||^6.0"
     },
     "require-dev": {
-        "openeuropa/code-review": "^0.2",
+        "openeuropa/code-review": "^0.3",
```
Comment (pfrenssen) [inline composer.json]: We don't need to state that we require 0.3 if we are not using the new features that are added in that version. We are still compatible with 0.2. A good idea would be to change the version range to `~0.2` then we are compatible with both 0.2, 0.3 and future versions.

**Comment by pfrenssen** [comment]
I really like Drone but their error messages suck. I have no idea what this means. This could be either a problem in the `.drone` file or on the server side configuration.

I see that a line of whitespace was removed from the Drone file, maybe we can reinstate that? If that doesn't help I would go to the devops team for help.

**Comment by imanoleguskiza** [comment]
I believe in this case this case the drone error is caused because the drone.yml file is not present in the branch yet.

**Comment by pfrenssen** [comment]
OK this makes sense. I see that the master branch is also failing with the same error. It is normally against our acceptance procedure to allow PRs with failing tests, but this is not causing a regression and providing Drone support is out of scope for this PR.

I think it is OK for me to accept this now. I have created an issue to take a look at Drone: https://github.com/ec-europa/oe-poetry-behat/issues/24

**Review by pfrenssen — APPROVED** [review]

**Comment by ku-enza** [comment]
Why do we have Travis and Drone here as CI tool?
Travis file do not exist in master branch...

**Comment by imanoleguskiza** [comment]
Because the branch was created before drone support was added. I think it should be enough to just remove the travis file.

**Review by ku-enza — APPROVED** [review]
Fixed the merge by removing .travis.yml,
:+1: 
Ready to be merged .

**Pull request merged**